src/System.Private.CoreLib/shared/System/DefaultBinder.cs

```diff
@@ -1204,6 +1204,63 @@ private static bool CreateParamOrder(int[] paramOrder, ParameterInfo[] pars, str
             return true;
         }
 
+        // CanChangePrimitive
+        // This will determine if the source can be converted to the target type
+        private static bool CanChangePrimitive(Type source, Type target)
+        {
+            Primitives widerCodes = s_primitiveConversions[(int)(Type.GetTypeCode(source))];
+            Primitives targetCode = (Primitives)(1 << (int)(Type.GetTypeCode(target)));
+
+            return (widerCodes & targetCode) != 0;
```

Comment:
Does it work correctly for `IntPtr` and `UIntPtr`?

Reply:
Good catch! CoreRT's impl doesn't.